requests_pkcs12.py

```diff
@@ -197,40 +222,23 @@ def selftest():
         "withEncryption": {
             "pkcs12_password": b"correcthorsebatterystaple",
             "expected_status_code": 200,
+            "expected_exception_message": None,
         },
         "withEmptyPassword": {
             "pkcs12_password": b"",
             "expected_status_code": 200,
+            "expected_exception_message": "Password must be 1 or more bytes.",
         },
         "withoutEncryption": {
             "pkcs12_password": None,
             "expected_status_code": 200,
+            "expected_exception_message": None,
         },
     }
+
     for test_case_name, test_case in test_cases.items():
-        password = test_case['pkcs12_password']
         print(f"Testing {test_case_name}")
```

Comment:
Please move the "print" also into the helper function, passing test_case_name as an additional parameter.